handbook/company/pricing-features-table.yml

```
@@ -326,6 +326,16 @@
       moreInfoUrl: https://docs.google.com/document/d/1qFYtMoKh3zyERLhbErJOEOo2me6Bc7KOOkjKn482Sqc/edit
     - description: See what servers someone is logged-in on
       moreInfoUrl: https://community.spiceworks.com/topic/138171-is-there-a-way-to-see-what-servers-someone-is-logged-in-on
+- industryName: Human-endpoint mapping (IdP groups)
+  description: Add user information (e.g Full name or group) from identity provider (e.g Okta) to your devices using SCIM protocol.
```

```suggestion
  description: Add user information (e.g full name and groups) from your identity provider (e.g Okta) to your devices using the SCIM protocol.
```

Small capitalization tweak and articles